app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,2 @@
+class LikesController < ApplicationController
+end
```

Comment:
- [ ] Kindly remove this likes controller, as there is no matching route for it.

Reply:
> * [ ]  Kindly remove this likes controller, as there is no matching route for it.

- [ ] Kindly remove this as requested by the previous reviewer.